### DISCUSSION TITLE
vnote 如何实现github同步笔记的操作

### DISCUSSION THREAD

**Discussion by GNknight** [tamlok/vnote#2498] [Q&A]
vnote 如何实现github同步笔记的便捷操作。我看菜单快捷键里由git  但是我找不到设置入口，从哪里设置？哪位大神教教或者出个攻略视频什么的

**Comment by tamlok** [comment]
Hi,

目前还没有内建的git支持，需要你把整个笔记本根文件夹通过github同步，快捷键里面的git入口只是方便调用外部程序，比如git commit之类的。
